olclient/openlibrary.py

```diff
@@ -196,8 +198,15 @@ def editions(self):
                 """
                 url = f'{self.OL.base_url}/works/{self.olid}/editions.json'
                 try:
-                    r = self.OL.session.get(url)
-                    editions = r.json().get('entries', [])
+                    r_json: Dict = self.OL.session.get(url).json()
```

Comment:
Thanks for providing the types as well but for container types we should provide their element types as well. In case of JSON data coming from the network, it's a bit difficult to get it right so for now we should use `Dict[Any, Any]`.

Reply:
Agreed!
Made changes @ https://github.com/internetarchive/openlibrary-client/pull/226/commits/f7c3b11183baf3ccaade8b892935d0bd0b3592ee includes changes for the other comments too.